render/src/umi/mod.rs

```diff
@@ -24,11 +29,51 @@ use unimarkup_parser::{
 
 pub mod render;
 
+fn unpack_content_safe(value: Value) -> String {
+    if value.value_type() == ValueType::Text {
+        value.as_str_or("").into()
+    } else {
+        value.as_cow_str_or("").into()
+    }
+}
+
+fn retrieve_localised_content(sheet: &Sheet, row_index: u32, col_index: u32) -> String {
+    let content_localised =
+        unpack_content_safe(sheet.cell(row_index, col_index).unwrap_or_default().value);
+    if content_localised.is_empty() {
+        unpack_content_safe(sheet.cell(row_index, 5).unwrap_or_default().value)
```

Comment:
```suggestion
        unpack_content_safe(sheet.cell(row_index, DEFAULT_CONTENT_COLUMN).unwrap_or_default().value)
```

Define a constant to get rid of magic numbers.
Because this is now used for fallback behavior, magic numbers make it less clear, why these columns are chosen.

For consistency, it might be better to use consts for all column numbers.